src/panels/lovelace/editor/card-editor/hui-card-editor.ts

```diff
@@ -33,7 +33,7 @@ declare global {
       config: LovelaceCardConfig;
       error?: string;
     };
-    "GUImode-changed": {};
+    "GUImode-changed": { guiMode: boolean };
```

Comment:
```suggestion
    "GUImode-changed": GUIModeChangedEvent;
```

Reply:
With this change how do I need to format 

```js
fireEvent(this as HTMLElement, "GUImode-changed", { guiMode });
```

```
Argument of type '{ guiMode: boolean; }' is not assignable to parameter of type 'GUIModeChangedEvent'.
  Type '{ guiMode: boolean; }' is missing the following properties from type 'GUIModeChangedEvent': bubbles, cancelBubble, cancelable, composed, and 18 more.
```

Reply:
Did you remove the `extends Event` from `GUIModeChangedEvent` like I suggested?

Reply:
You mean I should follow your suggestions?

Reply:
I think it could help 😆